firestore/tests/unit/v1/test_client.py

```diff
@@ -596,8 +596,9 @@ def test_found(self):
     def test_missing(self):
         ref_string = self._dummy_ref_string()
         response_pb = _make_batch_response(missing=ref_string)
+        reference_map = {ref_string: mock.sentinel.reference}
 
-        snapshot = self._call_fut(response_pb, {})
+        snapshot = self._call_fut(response_pb, reference_map)
```

Comment:
Add assertions here about the snapshot's `id`, data.

Reply:
Added assertions about snapshot's id and data @tseaver